### ISSUE TITLE
socket file path and name pretty hard coded

### ISSUE THREAD

**Issue by zlaio** [voltrue2/aeterno#1]
Hey,

I think socket file name and location is pretty hard coded in your library

`var sockName = '/tmp/' + (name || modName.get()) + '-monitor-' + path.replace(/\//g, '-') + '.sock';`

any chance to fix it using config or something?

**Comment by voltrue2** [comment]
Hello
  

  
Sure I can do that. Let me have a look.
  

  
Cheers
  

  
  

  
  
>

**Comment by zlaio** [comment]
Thanks.

I think, it would be nice if there is a config settings for this socket file (path+name). I noticed `aeterno` package also used in `gracenode`.

probably a method like `aeterno.setSocketName()` and a config settings in `gracenode` will be great

what do you think?

**Comment by voltrue2** [comment]
Yes. I think having both configurations and method(s) would be a good approach.
  

  
Cheers
  

  
  

  
  
>

**Comment by zlaio** [comment]
Thanks
